### PR TITLE
updates to nuopc cap

### PR DESCRIPTION
**Pull request by mvertens**
This PR only effects the NUOPC cap and brings in updates that accompany CMEPS updates.

### CODE REVIEW

File: src/cpl/nuopc/rof_comp_nuopc.F90 [hunk 1]
```diff
@@ -659,6 +647,18 @@ subroutine ModelAdvance(gcomp, rc)
        nlend = .false.
     endif
 
+    !--------------------------------
+    ! Unpack import state from mediator
+    !--------------------------------
+
+    call t_startf ('lc_mosart_import')
+
+    write(6,*)'DEBUG: calling mosart import'
```
Comment (ekluzek) [inline src/cpl/nuopc/rof_comp_nuopc.F90]: This should be removed, or put into a "if ( debug > 0 ) statement. Or the DEBUG string removed.

**Comment by mvertens** [comment]
@ekluzek - I have added the changes you requested and pushed back to github. Can this PR move ahead?

**Comment by ekluzek** [comment]
Yes. Did you run the one nuopc test or should I do that?

**Comment by mvertens** [comment]
@ekluzek - I am getting the same error with the correct tags. I will track this down. Thanks for doing this test.

**Comment by mvertens** [comment]
@ekluzek - I am getting the same error using the older code base. It seems to be dying in a pio call.

**Comment by mvertens** [comment]
@jedwards4b, @billsacks, @ekluzek - Backing up to pio1 fixed the problem - and the test ran successfully. 
This still had the modifications to the PR for `ncdio_pio.F90.in`. 
Its important to note that with mct as the coupling framework - the pio version out of the box is is pio1 **NOT** pio2. So this test will pass with pio1.  I have verified that `SMS_D_Vmct.f09_g17.I2000Clm50SpGs.cheyenne_intel` crashes the same way as `SMS_D_Vnuopc.f09_g17.I2000Clm50SpGs.cheyenne_intel `when pio2 is used.

**Comment by jedwards4b** [comment]
@mvertens Can we open a new issue regarding the failing test? SMS_D.f09_g17.I2000Clm50SpGs.cheyenne_intel (with mct and nuopc and pio2)
Can you tell if this a mosart issue or a CTSM one?

**Comment by mvertens** [comment]
@jedwards4b - this is a CTSM issue - not a MOSART issue. So this needs to
go into the CTSM issues and I believe is a "critical issue". I will open it
and assign it to you.

**Comment by billsacks** [comment]
@jedwards4b - I see you just pushed an additional fix to this branch. I'm going to start testing soon on this branch along with the corresponding nuopc_cap branches for rtm and ctsm. Do you plan to make any more changes to any of these branches?

**Comment by jedwards4b** [comment]
@billsacks I don't anticipate any further changes at this time.   Thanks!

**Pull request merged**